web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -225,6 +247,36 @@ export const InstantContactResolutionModel: ContactModel = {
   }
 }
 
+export function flickStartContactModel(params: GestureParams): ContactModel {
```

Comment:
```suggestion
export function FlickStartContactModel(params: GestureParams): ContactModel {
```

I am not actually sure which direction you want the casing to go because it looks like there are existing inconsistencies in this file. Fixup in the direction you think is best!

Reply:
Lowercase:  it's configured by a function.
Uppercase:  it's a completely static object.

Reply:
OK, that's a little obtuse :grin: `BasicLongpressContactModel` also doesn't seem to fit?

Reply:
I'll handle this and its related suggestions in #9998.